app/views/applications/index.html.haml

```diff
@@ -14,5 +14,8 @@
     %span.highlight It looks like something might be wrong. The latest application was received #{time_ago_in_words(@authority.latest_application_date)} ago.
     = link_to "Why?", faq_path(:anchor => "broken_scraper")
 
+- if current_user && @authority
+  = link_to "View as JSON", applications_json_path(@authority)
```

Comment:
I think this would go better above the pagination links at the end of the list. This is because this is a kinda nice to have and we don't know if people will really use it—so better to not add more links to the already overcrowded top of the page. I think put it down there and api users will discover it. We can also add click tracking to it to see if people do.